README.md

```diff
@@ -1,2 +1,33 @@
-# covid-19-projects
-A collaborative list of open source efforts in the fight against the covid-19 pandemic - explore.ovio.org/covid-19
+# COVID-19 Open Source Projects & Initiatives
+
+A collaborative list of open source efforts in the fight against the covid-19 pandemic. Ready to view on [explore/covid-19](https://explore.ovio.org/covid-19)
+
+![covid-19 issue](explore-covid-19.png)
+
+## Why is Ovio doing this?
+
+The world is facing an unprecedented crisis. People from all around the world have been coming together and are deploying an incredible amount of enerrgy to fight this pandemic. In just a few weeks, the open-source community has seeen thousands of projects and millions of contributors deploy their energy and skills.
+
+With everybody on our planet facing a commmon threat, open-source is a super-power, catalyzing the knowledge of everyone.
+
+Unfortunately, with so many individuals and groups hacking towwards similar goals, and despite working in the open, we are also witnessed an extraodrinary level of duplication.
+
+Our hope is to play our part in focusing efforts of developers, data-scientists and hackers to avoid duplication and maximize the impact key projects.
```

Comment:
Our hope is to play our part in focusing efforts of developers, data-scientists and hackers to avoid duplication and maximize the impact **of key** projects.